### DISCUSSION TITLE
Resizing table columns

### DISCUSSION THREAD

**Discussion by Minasyan-Gnel** [TanStack/table#2999] [Q&A]
I am trying to use column resizing feature in react-table` but could not.
I debugged and find that started from version 7.6.0 react-table lost resetResizing and getResizerProps functions.
Maybe something else is lost too.

I would be very grateful if you can fix that and help me out.

**Comment by Niks218** [comment]
I had the same problem, but turned out that I set `disableResizing` for first column. In this case there will be no `getResizerProps` on this column.
